arcade/examples/sound_demo.py

```diff
@@ -93,140 +88,43 @@ def setup(self):
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0, volume=volume)
-        button.center_x = SCREEN_WIDTH / 2
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4 * 3
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=1, volume=volume)
-        button.center_x = SCREEN_WIDTH - BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        y = SCREEN_HEIGHT / 2
-        volume = 0.5
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-1.0, volume=volume)
-        button.center_x = BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0, volume=volume)
-        button.center_x = SCREEN_WIDTH / 2
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4 * 3
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=1, volume=volume)
-        button.center_x = SCREEN_WIDTH - BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        y = SCREEN_HEIGHT / 2 - 50
-        volume = 1
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-1.0, volume=volume)
-        button.center_x = BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=-0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0, volume=volume)
-        button.center_x = SCREEN_WIDTH / 2
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=0.5, volume=volume)
-        button.center_x = SCREEN_WIDTH / 4 * 3
-        button.center_y = y
-        self.button_sprites.append(button)
-
-        button = SoundButton(":resources:sounds/upgrade4.wav", pan=1, volume=volume)
-        button.center_x = SCREEN_WIDTH - BUTTON_SIZE
-        button.center_y = y
-        self.button_sprites.append(button)
+        upgrade_sound = ":resources:sounds/upgrade4.wav"
+        volume_variation = [0.1, 0.5, 1]
+        y_offset = [50, 0, -50]
+        button_x_spread = [
```

Comment:
In its current form, this would be better put in a constant at the top of the file.